op-bindings/predeploys/addresses_test.go

```diff
@@ -26,18 +30,34 @@ func TestL1BlockSlots(t *testing.T) {
 	require.NoError(t, err)
```

Comment:
We need to disable this test because it's failed since op's bingings is different from us.
The test of this PR is not working because this PR has conflicts with dev.
```go
	t.Skip("tested in kroma-bindings")
```